api/management/commands/import-admin2-data.py

```diff
@@ -77,13 +71,18 @@ def handle(self, *args, **options):
 
       # loop through each feature in the shapefile
       for feature in data[0]:
-          code = feature.get("code")
-          name = feature.get("name")
+          code = feature.get("code") if 'code' in feature.fields else feature.get("pcode")
```

Comment:
@geohacker can there be a case where there's a `code` and a `pcode` and this would cause `pcode` to get dropped? Is that an issue? Should we store `pcode` as a separate field to make sure that is always a `pcode`?

Reply:
Ah good point. It is possible but our instructions are clear that it should be either code or pcode.